.drone.star

```diff
@@ -242,6 +242,16 @@ config = {
                 "GATEWAY_GRPC_ADDR": "0.0.0.0:9142",
             },
         },
+        "authApp": {
+            "suites": [
+                "apiAuthApp",
+            ],
+            "skip": False,
```

Comment:
```suggestion
            "skip": False,
            "withRemotePhp": [True],
```
I think it wouldn't be necessary to run this suite for both with and without remote.php since no dav path is involved.